tests/test_chat_to_files.py

```diff
@@ -1,7 +1,7 @@
 import textwrap
```

Comment:
I think we should arrange tests colder into core, cli, applications folders to match the source code